core/src/main/c/share/dedup.h

```diff
@@ -44,6 +44,24 @@ struct dedup_column {
     int64_t java_reserved_5;
     uint8_t null_value[32];
 };
+
+struct dedup_column_many_addresses {
```

Comment:
dedup_column_many_addresses has the same layout as dedup_column.  void *column_data; has same layout(can be casted) as void **column_data; 
you can also add generic accessors to dedup_column, like:
```
    template<typename T>
    T* get_column_data() {
        return reinterpret_cast<T*>(column_data);
    }

    template<typename T>
    T** get_column_data_2d() {
        return reinterpret_cast<T**>(column_data);  // Cast to a pointer to pointer
    }
```

Reply:
ok done in https://github.com/questdb/questdb/pull/5347/commits/cb3b691ca9ed97069f381816684378d92c5fe213